app/src/main/res/layout-land/view_suggested_edit_card.xml

```diff
@@ -0,0 +1,127 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Maybe I am wrong, but I think we don't need the landscape view for the card view since the card view will have almost the same ratio.

Reply:
Awesome catch! My eye had just gotten used to seeing two layouts..

Reply:
Done